velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -42,6 +42,15 @@ void registerAverageAggregate(
                              .argumentType(inputType)
                              .build());
   }
+
+  // Interval input type in Presto has special case and returns BIGINT, not
+  // DOUBLE.
+  signatures.push_back(exec::AggregateFunctionSignatureBuilder()
+                           .returnType("bigint")
```

Comment:
Should the return type also be interval type instead of just bigint?

Reply:
Good catch. I checked the Java implementation is writing the long value into an Interval, so the final type returned is an interval. I have updated this piece, can you re-review?